app_dart/lib/src/service/scheduler.dart

```diff
@@ -90,6 +138,11 @@ class Scheduler {
     return newCommits;
   }
 
+  /// Whether [Commit] already exists in [datastore].
```

Comment:
can you expand on this doc, explaining that this is cocoon's source of truth for builds it has scheduled, and that these builds could be scheduled via webhook OR cron job?

Reply:
Added!